src/Schema/AST/DocumentAST.php

```diff
@@ -88,6 +91,9 @@ class DocumentAST implements Arrayable
     /** @var array<int,SchemaExtensionNode> */
     public array $schemaExtensions = [];
 
+    /** A hash of the schema. */
+    public ?string $hash = null;
```

Comment:
Why is this nullable?

Reply:
For backward compatibility. In case someone is creating DocumentAST on their own, without filling the field.

Reply:
Not necessary, see https://github.com/nuwave/lighthouse?tab=readme-ov-file#versioning:

> However, only code elements marked with @api will remain compatible - all other code in Lighthouse is considered internal and is thus subject to change.

Reply:
ok, made it non-nullable